src/gatsby/src/utils/parkDatesHelper.js

```diff
@@ -77,17 +77,23 @@ const getFeatureDates = (dateArray) => {
     .filter(Boolean)
 }
 
-// Get park dates formatted
+// Get park dates formatted (current year only)
 const getParkDates = (parkOperationDates) => {  
   if (parkOperationDates.length === 0) {
     return ""
   }
 
-  // Format each date range and filter out empty ones
+  const currentYear = new Date().getFullYear()
+
+  // Format each date range, filter by operating year
   const formattedDateRanges = parkOperationDates
+    .filter(date => {
+      return date.operatingYear === currentYear &&
+             date.startDate && date.endDate
+    })
     .sort((a, b) => new Date(a.startDate) - new Date(b.startDate))
-    .map(dateData => formatDateRange(dateData.startDate, dateData.endDate))
-    .filter(dateStr => dateStr !== "")
+    .map(date => formatDateRange(date.startDate, date.endDate))
+    .filter(Boolean)
```

Comment:
What is `.filter(Boolean)` supposed to do?  Would there ever be non-truthy values like nulls in the array?